src/ophyd_async/core/_device/device_save_loader.py

```diff
@@ -0,0 +1,122 @@
+from enum import Enum
+from typing import Any, Dict, List
+
+import yaml
+from bluesky import Msg
+from numpy import ndarray
+
+from ophyd_async.core import Device, SignalRW
+
+
+def get_signal_RWs_from_device(
```

Comment:
I think PEP8 doesn't like uppercase in functions names. How about `walk_rw_signals(device, path_prefix)` instead?